packages/create-fuels/test/cli.test.ts

```diff
@@ -19,7 +19,7 @@ import { filterOriginalTemplateFiles, getAllFiles } from './utils/templateFiles'
 /**
  * @group node
  */
-describe('CLI', () => {
+describe('CLI', { timeout: 15_000 }, () => {
```

Comment:
👀

Reply:
You got me. I am guessing it is okay, though. Let's see.

This test was constantly failing; I experienced similar results with heavy-fs integration tests in the `fuels` CLI.

Reply:
We can rethink this strategy later :)